src/App.css

```diff
@@ -1,3 +1,197 @@
+  font-size: 0.875rem;
+  font-weight: bold;
+}
+
+.book-title {
+  width: 12.313rem;
+  height: 1.813rem;
+  margin: 0.188rem 0 0 0;
+  font-family: 'RobotoSlab', serif;
+  font-size: 1.375rem;
+  font-weight: bold;
+  letter-spacing: -0.2px;
+}
+
+.book-author {
+  width: 6.688rem;
+  height: 1.188rem;
+  margin: 0.25rem 0 0;
+  font-family: 'RobotoSlab', serif;
+  font-size: 0.875rem;
+  font-weight: 300;
+  color: #4386bf;
+}
+
+.book-link {
+  text-decoration: none !important;
+  font-family: 'RobotoSlab', serif;
+  font-size: 0.875rem;
+  font-weight: 300;
+  color: #4386bf;
+}
+
+.progress-oval {
+  position: relative;
+  border: 4px solid;
+  width: 4.25rem;
+  height: 4.25rem;
+  border-right-color: #307bbe;
+  border-left-color: white;
+  border-top-color: #307bbe;
+  border-bottom-color: white;
+  transform: rotate(30deg);
+  border-radius: 50%;
+}
+
+.percent-complete {
+  width: 4.25rem;
+  height: 2.438rem;
+  margin: 0.063rem 5.75rem 0 0.688rem;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 2rem;
+  font-weight: lighter;
+}
+
+.completed {
+  width: 5rem;
+  height: 1.125rem;
+  margin: 0.438rem 4.938rem 0.75rem 0.75rem;
+  opacity: 0.5;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 0.875rem;
+  font-weight: normal;
+}
+
+.current-chapter {
+  width: 7.875rem;
+  height: 1.125rem;
+  margin: 0.063rem 3.625rem 0 3.688rem;
+  opacity: 0.5;
+  font-family: 'RobotoSlab', serif;
+  font-size: 0.813rem;
+  font-weight: 300;
+}
+
+.current-lesson {
+  width: 4.688rem;
+  height: 1.313rem;
+  margin: 0.438rem 6.813rem 0.25rem 3.688rem;
+  font-family: 'RobotoSlab', serif;
+  font-size: 1rem;
+  font-weight: 300;
+  letter-spacing: -0.4px;
+}
+
+.btn-update {
+  width: 11.5rem;
+  height: 2.063rem;
+  margin: 0.75rem 0 0.563rem 3.688rem;
+  padding: 0.3rem 0.7rem 0.5rem 1rem !important;
+  border-radius: 3px;
+  background-color: #0290ff !important;
+  font-family: 'RobotoSlab', serif;
+  font-size: 0.813rem;
+  font-weight: 300;
+  letter-spacing: 0.5px;
+  text-align: center;
+  color: #fff !important;
+}
+
+.add-new-book {
+  width: 10.875rem;
+  height: 1.5rem;
+  margin: 1.813rem 32.75rem 1.188rem 0;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 1.25rem;
+  letter-spacing: -0.18px;
+  color: #888;
+}
+
+.add-book-form {
+  width: 75rem;
+  border-top: solid 2px #e8e8e8;
+  margin: auto;
+  margin-top: 20px;
+  padding-top: 20px;
+  margin-bottom: 50px;
+}
+
+.book-title-input {
+  width: 41.5rem;
+  height: 2.813rem;
+  padding: 0.813rem 1.063rem 0.813rem 1.063rem;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+}
+
+.book-category-input {
+  width: 17.813rem;
+  height: 2.813rem;
+  padding: 0.813rem 1.063rem 0.813rem 1.063rem;
+  margin: 1.188rem 2.125rem;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+}
+
+.add-book-button {
+  width: 11rem;
+  height: 2.813rem;
+  padding: 0.801rem 1.188rem 0.886rem 1.375rem;
+  border-radius: 3px;
+  background-color: #0290ff !important;
```

Comment:
- [ ] Please, don't use ! important tag, this is a bad practice and change the CSS flow on the project. 🌮